NEWS.md

```diff
@@ -1,3 +1,8 @@
+# mutscan 0.2.36
+
+* Check that reads don't exceed the maximal allowed length
+* Allow read lengths up to 65534 nt
```

Comment:
I guess here we could say `* Add parameter to specify maximal read length` instead of the second item

Reply:
Oops, yes, definitely - forgot to update this.